content/en-us/reference/engine/globals/LuaGlobals.yaml

```diff
@@ -499,6 +511,11 @@ functions:
 
       If `f` is `0`, then `setfenv()` changes the environment of the running
       thread and returns no values.
+
+      **WARNING:** This function allows uncontrolled change of the global/function
```

Comment:
Adding this up here; it's shown in the `deprecation_message:` field, but those are currently not rendering as expected.